rb/lib/selenium/webdriver/chrome/driver.rb

```diff
@@ -47,6 +47,12 @@ def execute_cdp(cmd, **params)
           @bridge.send_command(cmd: cmd, params: params)
         end
 
+        def print_page(**options)
+          options[:page_ranges] &&= Array(options[:page_ranges])
+
+          bridge.print_page(options)
```

Comment:
the bridge here isn't defined as an accessor / reader to try mask it better. So you need to directly call the iVar `@bridge` here.